modules/nf-core/dysgu/tests/main.nf.test

```diff
@@ -0,0 +1,96 @@
+nextflow_process {
+
+    name "Test Process DYSGU"
+    script "../main.nf"
+    process "DYSGU"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "dysgu"
+
+
+    test("human - bam") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test'], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.recalibrated.sorted.bam', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.recalibrated.sorted.bam.bai', checkIfExists: true)
+                ]
+                input[1] = [ [ id:'reference'], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("bam") },
```

Comment:
can you snapshot process.out.versions too?

Reply:
This is snapshoting process.out.versions @maxulysse .
But you don't need the "bam" in the match:
```suggestion
                { assert snapshot(process.out.versions).match() },
```

Reply:
:facepalm: that's what I meant, thanks for correcting me

Reply:
For me it always failed when we had multiple versions to assess. Thats why the snapshots were named.